nodejs/docs/node-standard-and-best-practices.md

```diff
@@ -109,6 +109,84 @@ For example:
 
 With the purpose of unifying the interfaces with other techs and making things easier to the client who may consume any other API, we decided to keep routes, input and output API parameters in **snake_case**.
 
+Some clarifications about the responses:
+
+If we need to return an object, for example an user, we must return it in this way using a key in the body.
```

Comment:
"If we need to return an object *that represents an entity*, for example a user, we must return it using a key in the body *that describes it as such*"